### DISCUSSION TITLE
Documentation?

### DISCUSSION THREAD

**Discussion by gregmaletic** [finnvoor/PlaydateKit#111] [General]
Sorry, basic question: it doesn't _appear_ to me that there is any documentation for the PlaydateKit APIs that are browsable in Xcode. Am I just not finding it, or does it need to be written? (I wrote a lot of the _Inside Playdate_ docs and would love to help out with that effort.)

**Comment by gregmaletic** [comment]
Ah, I did find this link, which seems like a decent start: https://finnvoor.github.io/PlaydateKit/documentation/playdatekit

**Comment by finnvoor** [comment]
~95% of the methods should be documented (mostly copy and pasted from the C docs), if you option-click one of them in Xcode you should see them:
![CleanShot 2024-12-25 at 18 48 17@2x](https://github.com/user-attachments/assets/667af7a3-ebab-4c6c-936d-0d2078e2fa7a)

You can also go to `Product > Build Documentation` in the menu bar and then you should be able to view all the docs in the Xcode Documentation browser:
![CleanShot 2024-12-25 at 18 27 01@2x](https://github.com/user-attachments/assets/5e753ff9-f743-445e-9ae1-ad0888045a0b)
